cmake/Thirdparty/GetGitRevisionDescription.cmake

```diff
@@ -0,0 +1,170 @@
+# - Returns a version string from Git
+#
```

Comment:
for each new file
if the code is copied/modified from others, need to copy the original license boilerplate. 
otherwise, put the Apache license V2 boilerplate.

Reply:
Yes. This is updated.